app/controllers/application_controller.rb

```diff
@@ -2,4 +2,9 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  before_filter :set_app_info_headers
+
+  def set_app_info_headers
+  	headers['X-GitSHA'] = AppInfo::GIT_REVISION
```

Comment:
OK maybe X-GIT-SHA or x-git-sha, headers are supposed to be case insensitive but the pattern seems to be either uppercase or lowercase.